pkg/resources/cloudcontroller/aws.go

```diff
@@ -142,9 +142,11 @@ func awsDeploymentCreator(data *resources.TemplateData) reconciling.NamedDeploym
 func getAWSCCMVersion(version semver.Semver) string {
 	switch version.MajorMinor() {
 	case v124:
+		return "v1.24.0"
+	case v125:
 		fallthrough
 	//	By default return latest version
 	default:
-		return "v1.24.1"
+		return "v1.25.0"
```

Comment:
there is v1.25.1 version of the registry.k8s.io/provider-aws/cloud-controller-manager